apps/server/src/api/admin/users/post.ts

```diff
@@ -1,17 +1,31 @@
 import { ulid } from 'ulid';
 import { fixedGroups } from '@raise/shared';
 import { middyfy } from '../../../helpers/wrapper';
-import { assertHasGroup, insert } from '../../../helpers/db';
+import { assertHasGroup, insert, scan } from '../../../helpers/db';
 import { userTable } from '../../../helpers/tables';
 import { $Ulid, $UserCreation } from '../../../schemas';
+import { sendEmail } from '../../../helpers/email';
+import newUser from '../../../helpers/email/newUser';
 
 export const main = middyfy($UserCreation, $Ulid, true, async (event) => {
   assertHasGroup(event, fixedGroups.National);
-
   const user = await insert(userTable, {
     id: ulid(),
     ...event.body,
   });
 
+  // finds name of the logged in user who made the user. If for some reason, the user's name is undefined, it will use The Raise National Team instead
+  const usersFromDb = await scan(userTable);
```

Comment:
nit: We actually only need to get this data inside the if branch - this would save us a costly database scan for when we don't send account emails.